leap_c/ocp/acados/parameters.py

```diff
@@ -0,0 +1,237 @@
+from typing import NamedTuple
+
+import casadi as ca
+import numpy as np
+from acados_template import AcadosOcp
+from casadi.tools import entry, struct, struct_symSX
+
+
+class Parameter(NamedTuple):
+    name: str
+    value: np.ndarray
+    # TODO: Check about infinity bounds.
+    lower_bound: np.ndarray | None = None
+    upper_bound: np.ndarray | None = None
+    fix: bool = True
+    differentiable: bool = False
+    stagewise: bool = False
```

Comment:
I would appreciate a docstring for this class.
I am not sure what exactly the `fix` entry is used for.